.travis.yml

```diff
@@ -30,6 +32,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install anybox.testing.openerp # needed for testing account_invoice_line_sort
```

Comment:
Why is [OCA/account-invoicing 8.0 build](https://travis-ci.org/OCA/account-invoicing/builds/67993591) green even without this package?